### ISSUE TITLE
Add a description field for incident types.

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#239]

**Comment by srabraham** [comment]
Good idea. This would be better than my suggestion in https://github.com/burningmantech/ranger-ims-server/issues/1413

**Comment by srabraham** [comment]
I like to take a staged approach here. As the first stage, I'd like to just link from the Incident page to the [relevant section in the wiki](https://github.com/burningmantech/ranger-ims-server/wiki/Incident-Types#ims-incident-types). That's trivially easy, and it gets us the important win of providing users with easily-updatable descriptions of the Incident Types.

The right long-term solution is what this issue describes, i.e. add a DESCRIPTION column to the INCIDENT_TYPE table, add some way to set/edit values for that column (on the admin incident types page?), and have some way for the UI to display all those descriptions (as a new page in the legacy web client?).

The long-term solution is a fair bit more involved, and less likely to be prioritized in the near-term. The link-to-the-wiki solution is a pretty good band-aid, with minimal downsides.

Any thoughts on that suggestion, @wsanchez?
